### ISSUE TITLE
fakeAsync is not working with webpack

### ISSUE THREAD

**Issue by frickt** [AngularClass/angular2-webpack-starter#590]
I've extended the app.spec.ts with a fakeAsync test.
```
import {
  it,
  inject,
  beforeEachProviders,
  fakeAsync,
  tick
} from '@angular/core/testing';
import {Control} from '@angular/common';

// Load the implementations that should be tested
import {App} from './app.component';
import {AppState} from './app.service';

describe('App', () => {
  // provide our implementations or mocks to the dependency injector
  beforeEachProviders(() => [
    AppState,
    App
  ]);

  it('should have a url', inject([ App ], (app) => {
    expect(app.url).toEqual('https://twitter.com/AngularClass');
  }));

  it("should rerun the validator when the value changes", fakeAsync(() => {
    var c = new Control("value");

    c.updateValue("expected");
    tick();

    expect(c.valid).toEqual(true);
  }));

});
```

and got the exception:
```
TypeError: undefined is not a constructor (evaluating 'new _FakeAsyncTestZoneSpecType()') in /Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js (line 23143)
    fakeAsync@/Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:23143:64 <- webpack:///~/@angular/core/testing/fake_async.js:24:0
    /Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:63422:91 <- webpack:///src/app/app.spec.ts:25:68
    /Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:63413:10 <- webpack:///src/app/app.spec.ts:14:9
    __webpack_require__@/Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:48:34 <- webpack:/webpack/bootstrap c3d4bc0d96fddcbb67fd:47:0
    webpackContext@/Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:63353:29 <- webpack:/src \.spec\.ts:10:0
    map@[native code]
    requireAll@/Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:153:36 <- webpack:///config/spec-bundle.js:56:0
    /Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:157:26 <- webpack:///config/spec-bundle.js:60:0
    /Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:159:35 <- null:0:0
    __webpack_require__@/Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:48:34 <- webpack:/webpack/bootstrap c3d4bc0d96fddcbb67fd:47:0
    /Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:91:37 <- webpack:/webpack/bootstrap c3d4bc0d96fddcbb67fd:90:0
    global code@/Users/thomasfrick/git/angular2-webpack-starter/config/spec-bundle.js:92:12 <- null:0:0

```

is there an import I miss?

**Comment by abdulhaq-e** [comment]
I can confirm this.

**Comment by abdulhaq-e** [comment]
Should be solved when #609 gets merged.

**Comment by gdi2290** [comment]
done via https://github.com/AngularClass/angular2-webpack-starter/pull/609
